docs/developer/testing/test_oadp_version_upgrade.md

```diff
@@ -26,6 +32,61 @@ Finally, after  creating a DPA, change channel of the installed operator to the
 
 ![Update subscription channel](../../images/test_oadp_version_upgrade_subscription.png)
 
+### From CLI
+
+Check if a OperatorGroup already exists in namespace you will install OADP operator
+```sh
+oc get operatorgroup -n <oadp-namespace>
+```
+If not, create one with this content
+```yaml
+kind: OperatorGroup
+apiVersion: operators.coreos.com/v1
+metadata:
+  name: oadp-operator-group
+  namespace: <oadp-namespace>
+spec:
+  targetNamespaces:
+    - <oadp-namespace>
+```
+For example, run
+```sh
+export OADP_NAMESPACE=openshift-adp
+echo -e "kind: OperatorGroup\napiVersion: operators.coreos.com/v1\nmetadata:\n  name: oadp-operator-group\n  namespace: $OADP_NAMESPACE\nspec:\n  targetNamespaces:\n    - $OADP_NAMESPACE" | oc create -f -
```

Comment:
there are many ways to create a yaml file with variables, this is fine. Not my preference but works :)